managed/models/grafana.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package models
+
+// GrafanaParams - defines flags and settings for grafana.
+type GrafanaParams struct {
```

Comment:
It looks more like PG params. Is it correct naming?

Reply:
these options are used to configure graphana datasource to use postgres. I suppose it is worth naming the parameters according to the place of their application and not about what they are. A matter of taste. It's not for me to make these decisions. If worth renaming ok for me.

Reply:
grafana params renamed to pg params